gulpfile.babel.js

```diff
@@ -0,0 +1,175 @@
+import fs from 'fs';
+import path from 'path';
+
+import gulp from 'gulp';
+
+// Load all gulp plugins automatically
+// and attach them to the `plugins` object
+import plugins from 'gulp-load-plugins';
+
+// Temporary solution until gulp 4
+// https://github.com/gulpjs/gulp/issues/355
+import runSequence from 'run-sequence';
+
+import archiver from 'archiver';
+import glob from 'glob';
+import del from 'del';
+
+import pkg from './package.json';
+
+const dirs = pkg['h5bp-configs'].directories;
+
+// ---------------------------------------------------------------------
+// | Helper tasks                                                      |
+// ---------------------------------------------------------------------
+
+gulp.task('archive:create_archive_dir', () => {
+    fs.mkdirSync(path.resolve(dirs.archive), '0755');
+});
+
+gulp.task('archive:zip', (done) => {
+
+    const archiveName = path.resolve(dirs.archive, `${pkg.name}_v${pkg.version}.zip`);
+    const zip = archiver('zip');
+    const files = glob.sync('**/*.*', {
+        'cwd': dirs.dist,
+        'dot': true // include hidden files
+    });
+    const output = fs.createWriteStream(archiveName);
+
+    zip.on('error', (error) => {
+        done();
+        throw error;
+    });
+
+    output.on('close', done);
+
+    files.forEach( (file) => {
+
+        const filePath = path.resolve(dirs.dist, file);
+
+        // `zip.bulk` does not maintain the file
+        // permissions, so we need to add files individually
+        zip.append(fs.createReadStream(filePath), {
+            'name': file,
+            'mode': fs.statSync(filePath).mode
+        });
+
+    });
+
+    zip.pipe(output);
+    zip.finalize();
+
+});
+
+gulp.task('clean', (done) => {
+    del([
+        dirs.archive,
+        dirs.dist
+    ]).then( () => {
+        done();
+    });
+});
+
+gulp.task('copy', [
+    'copy:.htaccess',
+    'copy:index.html',
+    'copy:jquery',
+    'copy:license',
+    'copy:main.css',
+    'copy:misc',
+    'copy:normalize'
+]);
+
+gulp.task('copy:.htaccess', () => {
+    return gulp.src('node_modules/apache-server-configs/dist/.htaccess')
+               .pipe(plugins().replace(/# ErrorDocument/g, 'ErrorDocument'))
+               .pipe(gulp.dest(dirs.dist));
+});
```

Comment:
@marcobiedermann This can be changed to something like:

```js
gulp.task('copy:.htaccess', () => 
    gulp.src('node_modules/apache-server-configs/dist/.htaccess')
        .pipe(plugins().replace(/# ErrorDocument/g, 'ErrorDocument'))
        .pipe(gulp.dest(dirs.dist));
);
```